### PR TITLE
feat: url for students

### PR DESCRIPTION
**Pull request by Mouwrice**
The students url is now `http://localhost:3000/students?id=3` when a student is selected. So if you reuse this link you will get student with id 3 but only and only if that student is on the first results page.

### CODE REVIEW

File: frontend/components/Students/Students.tsx [hunk 1]
```diff
@@ -57,6 +59,33 @@ export const Students: React.FC<{ alwaysLimited: boolean }> = ({
...
+     * Set the selected student
+     * from the url parameter
+     */
+    useEffect(() => {
+        const id = router.query.id;
+        if (id !== undefined) {
+            const id_number = Number(id);
+            if (!isNaN(id_number)) {
+                for (let i = 0; i < students.length; i++) {
+                    console.log(students[i].student.student_id);
```
Comment (BramDevlaminck) [inline frontend/components/Students/Students.tsx:72]: 🚨 get console.log policed!!! 🚨

**Review by BramDevlaminck — CHANGES_REQUESTED** [review]
Please remove the console.log, otherwise great job! Works as one would expect.

**Review by BramDevlaminck — APPROVED** [review]
awesome!

**Review by KoenDesplenter — APPROVED** [review]
Nicely done

**Pull request merged**